README.md

```diff
@@ -256,6 +258,52 @@ export default function () {
 }
 ```
 
+
+### Kinesis
+
+Utilize `Kinesis API` [Aws Reference page](https://docs.aws.amazon.com/kinesis/latest/APIReference/API_Operations.html) for more details on its methods, follow [example page](./examples/kinesis.js) on how to use.
```

Comment:
```suggestion
Consult the `KinesisClient` [dedicated k6 documentation page](https://k6.io/docs/javascript-api/jslib/aws/kinesisclient) for more details on its methods and how to use it.
```

I would prefer to follow the same sentence structure as the other examples. We will add the documentation page once this is released 👍🏻 🙇🏻

Reply:
done